nixpkgs/drv_set_builder.nix

```diff
@@ -0,0 +1,17 @@
+{ packages
+, nixpkgs ? import <nixpkgs> {}
```

Comment:
It's the default 99.9% of the times this file will be called, so I don't think it's worth bothering with passing it explicitly (esp. since everyone expects most top-level nix file to import <nixpkgs> anyways)